test/verify/check-packagekit

```diff
@@ -1076,6 +1076,24 @@ class TestUpdatesSubscriptions(PackageCase):
         b.wait_text(self.update_text, "Bug fix updates available")
         self.assertIn("fa-bug", b.attr(self.update_icon, "class"))
 
+    def testNoSubOsRepo(self):
+        m = self.machine
+        b = self.browser
+
+        # pretend we have a proper OS repo that does not require subscription
+        self.createPackage("coreutils", "999", "1")
```

Comment:
I guess we start in the "Not registered" state, right? The test would feel somewhat more complete, if it would also first check it is in "Not registered" state and then faking this repo solves the issue.

Reply:
We already have that in `TestUpdatesSubscriptions.testNoUpdates`. I didn't unify the tests, as that would first require some work in packagelib.py to be able to call enableRepo() twice (or some update function for it).

Reply:
Fair enough